src/pages/Appointments/components/PractitionerSelector.tsx

```
@@ -493,6 +500,31 @@ export const PractitionerSelector = ({
           {/* Sidebar Content */}
           <div className="flex-1 overflow-y-auto max-h-[400px]">
+            {/* Practitioner Search Inside Department */}
+            {currentOrganizationId && (
+              <div className="px-3 py-2 border-b bg-white sticky top-0 z-10">
+                <div className="relative flex items-center h-10">
+                  <SearchIcon className="h-4 w-4 absolute left-3 top-1/2 -translate-y-1/2 text-gray-500" />
+                  <input
+                    type="text"
+                    placeholder={t("search_practitioners")}
+                    value={practitionerSearch}
+                    autoFocus
```

Biome (2.1.2)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)